src/App.jsx

```diff
@@ -1,58 +1,114 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
 
+import cn from 'classnames';
+import { useState } from 'react';
+
+const SORT_FIELD = {
+  ALPHABET: 'name',
+  LENGTH: 'length',
+  NONE: 'none',
+};
+
 export const goodsFromServer = [
   'Dumplings',
   'Carrot',
   'Eggs',
-  'Ice cream',
   'Apple',
   'Bread',
   'Fish',
   'Honey',
   'Jam',
+  'Ice cream',
```

Comment:
Why did you reorder the initial data? 
maybe that's the reason why the test is failing.